workflows/qc/quality-check-standard.wdl

```diff
@@ -0,0 +1,155 @@
+    call ngsderive.readlen as ngsderive_readlen { input: bam=bam, max_retries=max_retries, wait_var=parse_input.input_check }
+
+    if (experiment == "RNA") {
+        call ngsderive.infer_strand as ngsderive_strandedness { input: bam=bam, bai=samtools_index.bai, gtf=gencode_gtf, max_retries=max_retries }
+        call qualimap.rnaseq as qualimap_rnaseq { input: bam=bam, gencode_gtf=gencode_gtf, provided_strand=provided_strand, inferred_strand=ngsderive_strandedness.strandedness, max_retries=max_retries }
+        call mqc.multiqc as multiqc_rnaseq {
+            input:
+                sorted_bam=bam,
+                validate_sam_string=validate_bam.out,
+                bigwig_file=deeptools_bamCoverage.bigwig,
+                qualimap_bamqc=qualimap_bamqc.results,
+                qualimap_rnaseq=qualimap_rnaseq.results,
+                fastqc_files=fastqc.out_files,
+                fastq_screen=fastq_screen.out_files,
+                flagstat_file=samtools_flagstat.outfile,
+                max_retries=max_retries
+        }
+    }
+    if (experiment == "WGS" || experiment == "WES") {
+        call mqc.multiqc {
+            input:
+                sorted_bam=bam,
+                validate_sam_string=validate_bam.out,
+                bigwig_file=deeptools_bamCoverage.bigwig,
+                qualimap_bamqc=qualimap_bamqc.results,
+                fastqc_files=fastqc.out_files,
+                fastq_screen=fastq_screen.out_files,
+                flagstat_file=samtools_flagstat.outfile,
+                max_retries=max_retries
+        }
+    }
+
+    output {
+        File bam_checksum = compute_checksum.outfile
+        File bam_index = samtools_index.bai
+        File flagstat = samtools_flagstat.outfile
+        Array[File] fastqc_results = fastqc.out_files
+        File qualimap_bamqc_results = qualimap_bamqc.results
+        File? qualimap_rnaseq_results = qualimap_rnaseq.results
+        Array[File] fastq_screen_results = fastq_screen.out_files
+        File bigwig = deeptools_bamCoverage.bigwig
+        File? multiqc_zip = multiqc.out
+        File? multiqc_rnaseq_zip = multiqc_rnaseq.out
+        File? inferred_strandedness = ngsderive_strandedness.strandedness_file
+        File instrument_file = ngsderive_instrument.instrument_file
+        File readlen_file = ngsderive_readlen.readlen_file
+    }
+}
+
+task parse_input {
+    input {
+        String input_experiment
+        String input_strand
+        File input_fq_db
+        String input_fq_format
+    }
+
+    command {
+        if [ "~{input_experiment}" != "WGS" ] && [ "~{input_experiment}" != "WES" ] && [ "~{input_experiment}" != "RNA" ]; then
+            >&2 echo "experiment input must be 'WGS', 'WES', or 'RNA'"
+            exit 1
+        fi
+
+        if [ -n "~{input_strand}" ] && [ "~{input_strand}" != "stranded-reverse" ] && [ "~{input_strand}" != "stranded-forward" ] && [ "~{input_strand}" != "unstranded" ]; then
```

Comment:
Let's make all of these capitalized (`Stranded-Reverse`) to mirror what we do with `ngsderive`.